docker/_nix-build

```diff
@@ -37,11 +37,14 @@ def loadimage(image_archive_filename):
 
 def main(names):
     for name in names:
-        print("Building {}.".format(name))
+        report("Building {}...\n".format(name))
         result = nixbuild(name.decode("utf-8"))
-        print("Build complete.  Loading {}.".format(result))
+        report("Build complete.  Loading {}...\n".format(result))
         loadimage(result)
-        print("Load complete.")
+        report("Load complete.\n")
 
+def report(s):
+    stdout.write(s)
```

Comment:
`print()` takes a `flush=` kwarg, so could just be `print(s, flush=True)` I believe ...